File: acm_statistic_backend/api/api/DTO/evolutionTemporel/ComparaisonDynamiqueResumerDTO.cs
namespace api.DTO.evolutionTemporel
{
    public class ComparaisonDynamiqueResumerDTO
    {
        public decimal? Annee_actuelle { get; set; }
        public decimal? Annee_precedente { get; set; }
        public decimal? Variation {  get; set; }
    }
}

File: acm_statistic_frontend/Interface Tableau de Bord Analytique Aérien/src/models/evolutionTemporel/ComparaisonDynamiqueResumerDTO.ts
export class ComparaisonDynamiqueResumerDTO{
    annee_actuelle: number;
    annee_precedente: number;
    variation: number | null;

    constructor(init?: Partial<ComparaisonDynamiqueResumerDTO>) {
    Object.assign(this, init);
    }

}